content/200-concepts/100-components/02-prisma-client/190-deployment.mdx

```diff
@@ -74,18 +74,45 @@ postgresql://USER:PASSWORD@HOST:PORT/DATABASE?connection_limit=1
 mysql://USER:PASSWORD@HOST:PORT/DATABASE?connection_limit=1
 ```
 
+### Usage with PgBouncer
+
+Typically PostgreSQL supports only a certain amount of concurrent connections and this limit can be reached quite fast when the service usage goes up – especially in serverless environments.
+
+A connection pooler like PgBouncer prevents your application from exhausing the database's connection limit.
+
+PgBouncer holds a connection pool to the database and proxies incoming client connections by sitting between Prisma Client and the database.
+
+PgBouncer funnels client connections into a pooling application, which reduces the number of processes a database has to handle at any given time. PgBouncer passes on a limited number of connections to the database and queues additional connections for delivery when space becomes available.
+
+To use Prisma Client with PgBouncer from a serverless function, add the `?pgbouncer=true` flag to the PostgreSQL connection string and set the connection limit to `1`:
```

Comment:
Setting Connection Limit to 1 is unrelated to usage of PgBouncer. It is a thing you can do in specific circumstances (when you are running on serverless with an application that never uses more than 1 connectino in parallel or you really want to optimize on the number of serverless functions that can run in parallal)